### ISSUE TITLE
Adding COCOB to TF contrib

### ISSUE THREAD

**Issue by Faur** [bremen79/cocob#1]
I was just wondering if you will try to add this to the official TF repo?

**Comment by bremen79** [comment]
Yes, I will. I will have a couple of talks at Google in the next months, it will be a good occasion to ask them what is the best way to do it.

**Comment by stsievert** [comment]
I’d also like to see this. Pulling from the [tensorflow/contrib readme][1],

> When adding a project, please stick to the following directory structure: Create a project directory in contrib/, and mirror the portions of the TensorFlow tree that your project requires underneath contrib/my_project/.

[1]:https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/README.md

**Comment by mihahauke** [comment]
@bremen79 Is there any progress or hope of progress with putting Cocob into TF contrib?

**Comment by bremen79** [comment]
Hi All, I would also like to see Cocob in TF, but I am not sure I have time to do it...
Anyone can give me a hand to do it?

**Comment by mihahauke** [comment]
@bremen79 I could try but I am not sure what about my time in the nearest future so I can't promise anything.
